Coalition Folklore: Arachi (#5898)
endless-sky/endless-sky: data/coalition/coalition missions.txt

			`	As the rows and balconies fill up with more guests, you notice a few Kimek and Saryds amidst the masses of Arachi, most of them accompanying an Arach patron, but a few seemingly being patrons themselves, coming in on their own. A great many of them seem to take notice of you as you look around, but your patron studiously ignores the attention. Once all seats are filled, the doors are closed, and an Arach comes to the stage. "Welcome you, esteemed guests, to this celebration of saga and song, our theater does," he says. "Make you wait any longer, I shall not, so for our first play, a timeless classic dear to all, I present: Moonstruck."`

> 			`	He quickly leaves the stage as the curtains open to reveal a rocky wall, with two tunnels leading to the sides. The orchestra begins to play a slow, soft theme on accordion-like instruments, and a young Arach comes into the scene from the tunnel on the left. He hangs onto the rocky wall, a small lantern hanging from his abdomen, and a large sack hanging from his back. He picks rocks from the corners of the wall, throwing most to the ground, and placing a few inside the sack he carries. Once the sack is full, he enters the tunnel on the right, and the rocky wall is rotated to show the other side, where the young miner is shown outside the mine, under the night sky, smelting the ore he collected. As he looks to the top of the stage, his back facing the audience, the accordions grow quieter, and a slow harp starts to play, as a full moon comes into view, at the top of the background sky.`

Reviewer: ```suggestion
			`	He quickly leaves the stage as the curtains open to reveal a rocky wall with two tunnels leading to the sides. The orchestra begins to play a slow, soft theme on accordion-like instruments, and a young Arach comes into the scene from the tunnel on the left. He hangs onto the rocky wall, a small lantern hanging from his abdomen and a large sack strapped to his back. He picks rocks from the corners of the wall, throwing most to the ground and placing a few inside the sack he carries. Once the sack is full, he enters the tunnel on the right, and the rocky wall is rotated to show the other side, where the young miner is shown under the night sky smelting the ore he collected. As he looks to the top of the stage, his back facing the audience, the accordions grow quieter, and a slow harp starts to play as a full moon comes into view at the top of the background sky.`
```
Just some commas and a couple small things: avoiding using `hanging` twice in the same sentence and taking `outside the mine` out (sort of redundant since `under the night sky` would imply we're outside the mine now anyways).

> 			`	The curtains close as the scene changes, opening again with the accordions picking up the pace again, with the Arach now on a larger wall, with only one tunnel from above. He claws away at the rocky corners, the accordions following his rhythm. Once he's dug out most of the ore, the wall starts shaking, as if the mine were collapsing, and the accordions become erratic as he rushes for the escape tunnel. With a hectic clamour the rock wall appears to disassemble on stage with great boulders descending gracefully from the fly tower, "caving-in" the mine, and the accordions stop. After a long pause they slowly pick up again, along with the harp, as the young Arach climbs into view and pushes first one boulder away, and then another, and another, until the Arach stands atop a mountain of metal carved out of the rock, reaching for the moon. As he nearly touches it, the curtains close, and the music stops.`

Reviewer: ```suggestion
			`	The curtains close as the scene changes, opening again with the accordions picking up the pace. The Arach is now on a larger wall with only one tunnel from above. He claws away at the rocky corners, the accordions following his rhythm. Once he's dug out most of the ore, the wall starts shaking, as if the mine were collapsing, and the accordions become erratic as he rushes for the escape tunnel. With a hectic clamor, the rock wall appears to disassemble on stage with great boulders descending gracefully from the fly tower, caving-in the mine, and the accordions stop. After a long pause, they slowly pick up again, along with the harp, as the young Arach climbs into view and pushes first one boulder away, and then another, and another, until the Arach stands atop a mountain of metal carved out of the rock, reaching for the moon. As he nearly touches it, the curtains close, and the music stops.`
```

> 			`	The Arachi in the audience rub two of their legs together, as if clapping at the end of the performance. Shortly after, some get up, heading to the exit, as the doors open up once again. At the same time, more patrons are coming in to fill the vacant seats. "Interested in only some of the plays, some patrons are, so only for those they wish to see, they stay," the Arach beside you says. "Leave after any of the performances, you may, if someplace else, you must be, but remain for the rest of the show, I will."`

Reviewer: ```suggestion
			`	The Arachi in the audience rub two of their legs together, as if clapping at the end of the performance. Shortly after, some get up, heading to the exit as the doors open up once again. At the same time, more patrons are coming in to fill the vacant seats. "Interested in only some of the plays, some patrons are, so only for those they wish to see, they stay," the Arach beside you says. "Leave after any of the performances, you may, if someplace else, you must be, but remain for the rest of the show, I will."`
```

			choice
				`	(Stay for the next play.)`
				`	(Leave.)`
					goto leaveearly
			`	Once everyone is seated, the same Arach from before goes to the stage again. "A surprise for you, tonight we have. A brand new play, an anonymous group of patrons commissioned. As a critique of one of the issues that persist in our society, described it is. For our second play: Babbling Bebliss."`

> 			`	As he leaves, the curtains open to a set of an Arachi street, where a child is purchasing some form of candy. From behind a building in the set, comes an Arach wearing tall, blue and white "socks," as trumpets and tubas start playing an obnoxious track. He runs for the child and knocks the candy to the ground, as he makes some gargling sound. The Arach beside you sighs, saying that "Gibberish, it is. Nonsense, and words best left untranslated." The Arach actor goes on to cling to the walls of the building and grab things from inside the candy shop, throwing them wildly. By now, many well dressed patrons, most likely members of house Bebliss, are getting up and leaving the theater. Meanwhile, a group of Arachi on one of the balconies doesn't even try to muffle their laughter.`

Reviewer: ```suggestion
			`	As he leaves, the curtains open to a set of an Arachi street, where a child is purchasing some form of candy. From behind a building in the set, an Arach wearing tall, blue and white "socks" emerges as trumpets and tubas start playing an obnoxious track. He runs for the child and knocks the candy to the ground as he makes some gargling sound. The Arach beside you sighs, saying, "Gibberish, it is. Nonsense, and words best left untranslated." The Arach actor goes on to cling to the walls of the building and grab things from inside the candy shop, throwing them wildly. By now, many well dressed patrons, most likely members of house Bebliss, are getting up and leaving the theater. Meanwhile, a group of Arachi on one of the balconies doesn't even try to muffle their laughter.`
```

> 			`	The scene continues, as a couple comes to the set, walking down the street. As they're about to do what looks like a kiss, the Arach leaps from the building, landing on the bachelor and stealing the kiss, at which point the tubas and trumpets blast as loudly as they can. After a while, the authorities show up, only for the Arach to make a run for it down the street, as the orchestra plays without coordination, and the curtains close. Not many in the audience manifest themselves, most sighing or getting ready to leave, save for the group that was laughing earlier, who are very vocal in announcing their appreciation for the play. "Know of subtetly, house Chamgar does not," the Arach beside you says. "A shame it is, that tainted the soiree with this rivalry, they have."`

Reviewer: ```suggestion
			`	The scene continues as a couple comes to the set, walking down the street. As they're about to do what looks like a kiss, the Arach leaps from the building, landing on the bachelor and stealing the kiss, at which point the tubas and trumpets blast as loudly as they can. After a while, the authorities show up, only for the Arach to make a run for it down the street as the orchestra plays without coordination and the curtains close. Not many in the audience manifest themselves, most sighing or getting ready to leave, save for the group that was laughing earlier, who are very vocal in announcing their appreciation for the play. "Know of subtetly, house Chamgar does not," the Arach beside you says. "A shame it is, that tainted the soiree with this rivalry, they have."`
```

> 			`	The doors open once again, as new patrons cycle in to watch the third play.`

Reviewer: ```suggestion
			`	The doors open once again as new patrons cycle in to watch the third play.`
```

			choice
				`	(Stay for the next play.)`
				`	(Leave.)`
					goto leaveearly
			`	Everyone takes their seats, and the Arach announcer heads to the stage again. "A cathartic tragedy, our third play of this evening is. With you: 'Til Death."`

> 			`	This time, all lights inside the theater are turned off, as the curtains open to a single spotlight, on the right side of the stage, highlighting an old Arach inside a small shack, getting dressed. He coughs, and a piano begins to play a melancholic melody. Once he leaves the shack, it's left in the dark, as the spotlight focuses on the middle of the stage, an empty street. Once more, the spotlights shift focus, finally lighting up the left side of the stage, where the Arach starts tending to a garden by the entrance of a manor. The piano stops its weeping every once in a while, right when the Arach is coughing, only to resume once he's done. Leaving the manor, a richly dressed Arach points to one of the flowers in the garden, showing it's shriveled and dead, plucking it and throwing it to the ground before leaving. The gardener picks up the flower, sticking it inside his clothes, and continues his work.`

Reviewer 2: Theatre lights are usually turned off, the stage lights would be the relevant ones. I've also swapped it out to an organ approximation. An actual piano, or something too similar to it would be almost, if not impossible to play for an Arach, but a common joke about an organ is that it needs 4 feet to play it.  Plus some general stagecraft alterations and grammar.
```suggestion
			`	The stage lights go dark as the curtains open to a single spotlight. On the right side of the stage it illuminates an old Arach getting dressed inside a small shack. He coughs, and a something akin to an organ begins to play a melancholic melody. Once he leaves the shack it's left in the dark, as the spotlight follows him to the middle of the stage where he makes his way carefully down an empty street. He is allowed to pass beyond the spotlight which goes dark as the perspective shifts and a second spotlight renews the focus, finally lighting up the left side of the stage where the Arach starts tending to a garden by the entrance of a manor. The piano stops its weeping every once in a while, right when the Arach is coughing, only to resume once he's done. Leaving the manor, a richly dressed Arach points to one of the flowers in the garden, showing it to be shriveled and dead, plucking it and throwing it to the ground before leaving. The gardener picks up the flower, sticking it inside his clothes, and continues his work.`
```

> 			`	The stage lights go dark as the curtains open to a single spotlight. On the right side of the stage it illuminates an old Arach getting dressed inside a small shack. He coughs, and a something akin to an organ begins to play a melancholic melody. Once he leaves the shack it's left in the dark, as the spotlight follows him to the middle of the stage where he makes his way carefully down an empty street. He is allowed to pass beyond the spotlight which goes dark as the perspective shifts and a second spotlight renews the focus, finally lighting up the left side of the stage where the Arach starts tending to a garden by the entrance of a manor. The piano stops its weeping every once in a while, right when the Arach is coughing, only to resume once he's done. Leaving the manor, a richly dressed Arach points to one of the flowers in the garden, showing it to be shriveled and dead, plucking it and throwing it to the ground before leaving. The gardener picks up the flower, sticking it inside his clothes, and continues his work.`

Reviewer: ```suggestion
			`	The stage lights go dark as the curtains open to a single spotlight. On the right side of the stage it illuminates an old Arach getting dressed inside a small shack. He coughs, and a something akin to an organ begins to play a melancholic melody. Once he leaves, the shack is left in the dark as the spotlight follows him to the middle of the stage where he makes his way carefully down an empty street. He is allowed to pass beyond the spotlight which goes dark as the perspective shifts and a second spotlight renews the focus, finally lighting up the left side of the stage where the Arach starts tending to a garden by the entrance of a manor. The organ stops its weeping every once in a while, right when the Arach is coughing, only to resume once he's done. Leaving the manor, a richly dressed Arach points to one of the flowers in the garden, showing it to be shriveled and dead, plucking and throwing it to the ground before leaving. The gardener picks up the flower, sticking it inside his clothes, and continues his work.`
```

> 			`	Once his work is done, he heads back to his shack, the spotlights following him as he moves around the stage. Once inside his shack, he looks inside a small box, taking a simple vase, and placing the flower inside it. The stage goes dark as he goes to sleep, and the piano stops, only the occasional coughing fit echoing throughout the theater. When the light comes to the shack again, the Arach slowly gets up, and leaves for the gardening work again, pausing for a moment on the street part of the stage during a heavy coughing fit. Once he's done with the work again, and begins going back home, he starts coughing again, not letting up for more than a second at a time, during which the spotlights transition to the next section of the stage, and the piano plays a brief note. Once inside his shack, he collapses, knocking down the flower vase in the process. The piano plays a few slow notes, and the last spotlight goes out during the final note.`

Reviewer: ```suggestion
			`	Once his work is done, he heads back to his shack, the spotlights following him as he moves around the stage. Once inside, he looks inside a small box, taking a simple vase, and placing the flower inside it. The stage goes dark as he goes to sleep, and the piano stops, only the occasional coughing fit echoing throughout the theater. When the light comes to the shack again, the Arach slowly gets up and leaves for the gardening work again, pausing for a moment on the street part of the stage during a heavy coughing fit. Once he's done with the work again and begins going back home, he starts coughing again, not letting up for more than a second at a time, during which the spotlights transition to the next section of the stage, and the piano plays a brief note. Once inside his shack, he collapses, knocking down the flower vase in the process. The piano plays a few slow notes, and the last spotlight goes out during the final note.`
```

> 			`	You hear the curtains closing, and the lights of the theater turn back on again, as the audience is taken by a round of applause. Once they've settled, the announcer goes up to the stage again. "And with that, concluded, our soiree is. Thank you all for gracing us with your presence, I, in name of the theater, do." He lowers his head in a bow, and leaves the stage again, as nearly all of the patrons in the audience get up and head for the exit, this time with nobody else coming in to take their seats. At the same time, the orchestra starts packing up their instruments. There are, however, a few who remain in their seats, including the Arach who brought you here.`

Reviewer: ```suggestion
			`	You hear the curtains closing, and the lights of the theater turn back on again as the audience is taken by a round of applause. Once they've settled, the announcer goes up to the stage again. "And with that, concluded, our soiree is. Thank you all for gracing us with your presence, I, in name of the theater, do." He lowers his head in a bow and leaves the stage again as nearly all of the patrons in the audience get up and head for the exit, this time with nobody else coming in to take their seats. At the same time, the orchestra starts packing up their instruments. There are, however, a few who remain in their seats, including the Arach who brought you here.`
```

			choice
				`	(Stay.)`
					goto end

> 				`	(Leave.)`

Reviewer: Does this need a different branch than just `leaveearly`? In this case, there's not really many other Arachi patrons to watch you go.

			label leaveearly
			`	You thank the Arach beside you for inviting you here, and while she doesn't seem to pay you much mind, you can feel the gazes of other Arachi patrons on you, whispering among themselves as you leave. A few minutes after leaving, you're back at your ship.`
				decline
			label end

> 			`	The theater workers leave one after the other, leaving you and the other guests with an empty stage. After a while, the lights are turned off. A couple minutes later, however, you hear the curtains open, as some violins start to play in unison, not from where the orchestra was seated, but as if they were all over the theater, behind you and even up on the balconies. A strong light comes from the stage, most of it blocked by a tall, dark figure, but it isn't an actor, seemingly being made of some sort of wooden parts, kept up by thin wires you can barely spot when looking at the top of the stage. The violins' song is disrupted by dull, successive drum hits, at which point the figure moves, with "arms" opening to the sides and becoming even taller as it stands up. It is a humanoid creature, and it begins to slowly move around the stage, as if dancing in a pendulum. Cellos join in with the violin screams, and are followed by a choir, though you cannot understand what they said. You do recognize, however, that it isn't being sung in any language of the Coalition.`

Reviewer 3: ```suggestion
			`	The theater workers leave one after the other, leaving you and the other guests with an empty stage. After a while, the lights are turned off. A couple minutes later, however, you hear the curtains open, as some violins start to play in unison. Their music doesn't come from where the orchestra was seated, but as if they were all over the theater, behind you and even up on the balconies. A strong light comes from the stage, most of it blocked by a tall, dark figure. However, it isn't an actor, instead seemingly made out of wooden parts kept up by thin wires you can barely spot when looking at the top of the stage. The violins' song is disrupted by dull, successive drum hits, at which point the figure moves, with its "arms" opening to the sides; it becomes even taller as it stands up. It is a humanoid creature, and it begins to slowly move around the stage, as if dancing in a pendulum. Cellos join in with the violin screams, and are followed by a choir, though you cannot understand what they said. You do recognize, however, that it isn't being sung in any language of the Coalition.`
```

> 			`	The theater workers leave one after the other, leaving you and the other guests with an empty stage. After a while, the lights are turned off. About ten minutes later, however, you hear the curtains open, as some violins start to play in unison, not from where the orchestra was seated, but as if they were all over the theater, behind you and even up on the balconies. A strong light comes from the stage, most of it blocked by a tall, dark figure, but it isn't an actor, seemingly being made of some sort of wooden parts, kept up by thin wires you can barely spot when looking at the top of the stage. The violins' song is disrupted by dull, successive drum hits, at which point the figure moves, with "arms" opening to the sides, and becoming even taller, as it stands up. It is a humanoid creature, and it begins to slowly move around the stage, as if dancing in a pendulum. Cellos join in with the violin screams, and are followed by a choir, though you cannot understand what they said. You do recognize, however, that it isn't being sung in any language of the Coalition.`

Reviewer: ```suggestion
			`	The theater workers leave one after the other, leaving you and the other guests with an empty stage. After a while, the lights are turned off. A couple minutes later, however, you hear the curtains open, as some violins start to play in unison, not from where the orchestra was seated, but as if they were all over the theater, behind you and even up on the balconies. A strong light comes from the stage, most of it blocked by a tall, dark figure, but it isn't an actor, seemingly being made of some sort of wooden parts, kept up by thin wires you can barely spot when looking at the top of the stage. The violins' song is disrupted by dull, successive drum hits, at which point the figure moves, with "arms" opening to the sides and becoming even taller as it stands up. It is a humanoid creature, and it begins to slowly move around the stage, as if dancing in a pendulum. Cellos join in with the violin screams, and are followed by a choir, though you cannot understand what they said. You do recognize, however, that it isn't being sung in any language of the Coalition.`
```
10 minutes seems like a very long time for someone to sit in a dark theater and not ask any questions. Would it be problematic to change this? If so, maybe we could add something where the player tries to ask a question and is answered with a "just wait and see".

> 			`	The creature appears to get closer to the audience, growing in size, and as the choir and instruments  more become intense, it swings one of its arms upward, removing the wires. It stays upright, however, and continues to move. It bring both arms close and strikes its own head to form two small holes from which the background light shines through. It falls to its knees, with one arm reaching for the audience, as the light behind it turns off, and the choir and music subside.`

Reviewer 3: ```suggestion
			`	The creature appears to get closer to the audience, growing in size, and as the choir and instruments become more intense, it swings one of its arms upward and removes the wires. Despite this, it stays upright and continues to move. It brings both arms close and strikes its own head to form two small holes from which the background light shines through. Then, it falls to its knees, with one arm reaching for the audience, as the light behind it turns off and the choir and music subside.`
```

> 			`	The creature appears to get closer to the audience, growing in size, and as the choir and instruments grown more intense, it swings one of its arms upward, removing the wires. It stays upright, however, and continues to move, now bring both arms close, and striking its own head, forming two small holes from which the background light shines through. It falls to its knees, with one arm reaching for the audience, as the light behind it turns off, and the choir and music subside.`

Reviewer: ```suggestion
			`	The creature appears to get closer to the audience, growing in size, and as the choir and instruments  more become intense, it swings one of its arms upward, removing the wires. It stays upright, however, and continues to move. It bring both arms close and strikes its own head to form two small holes from which the background light shines through. It falls to its knees, with one arm reaching for the audience, as the light behind it turns off, and the choir and music subside.`
```
Just trying to improve the flow and clarity a bit.

> 			`	The lights turn back on, revealing curtains still open, but nothing on the stage. A few workers come in to tidy up the place, and the remaining guests now get up to leave. "Enjoyed the soiree, have you, Captain?" The Arach beside you asks, as she herself gets up to leave. "Come now, actually over, the show now is." You follow her outside, and she guides you back to your ship. "Thankful for you accompanying me, I am, Captain. Safe travels, may you have."`

Reviewer: I think the player should have the chance to ask what the heck it was they just saw. To be honest, there doesn't even have to be a substantial answer, or you can have the Arach avoid the question, but I would say let the player at least ask. That's a really strange experience; it would feel weird to just accept and move on with your day.
Also - what's the figure actually supposed to be? Some strange cultural thing? Foreshadowing for something in the Heliarch/Resistance campaigns? Just really curious about this myself.

Author: The figure is meant to be a Quarg, and the choir is in a language the player doesn't recognize for it is sung in the Quarg language.
After the events of the Quarg war, the Coalition would've gotten a boogeyman story of sorts related to the Quarg, this was an Arach take on the tale, being one of the plays.

I'll try and add a thing later about the player trying to ask about it.

				accept
			label noship
			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`

mission "Coalition Folklore: Arach"
	minor
	invisible
	to offer
		random < 30
	source
		attributes "arach" "folklore"
	on offer
		conversation

> 			`Shortly after entering the spaceport, you observe an Arach in what you guess is a formal "dress," a delicate silk covering most of her body with each leg covered by its own embroidered skirt. She exclaims something loudly into a personal communication device before putting it away with a shove. Like everyone else who has momentarily been witness to this, you go to look away, but the Arach has cast their gaze around, seen you, and immediately begun to approach. "Our honorable human visitor, I salute," she says loudly as she closes the gap. "Say, Captain, appreciate the fine arts, do you? To a cultural soiree, I'm headed, but busy with work, dearest Papa is, so vacant, his reserved seat is. Accompany me, would you?"`

Author: As Grey said, the idea is that the patrons attend the soiree as a show of status, and bringing exotic guests is a "powermove" of sorts. As her father couldn't attend, she sees this as an opportunity to bring a very exotic guest, seeing you nearby.
The screaming in the cellphone part would be more a show of anger at her father, for presumably cancelling his presence right when the event is about to start.

> 			`Shortly after entering the spaceport, you observe an Arach in what you guess is a formal "dress," a delicate silk covering most of her body with each leg covered by its own embroidered skirt. She exclaims something loudly into a personal communication device before putting it away with a shove. Like everyone else who has momentarily been witness to this, you go to look away, but the Arach has cast their gaze around, seen you, and immediately begun to approach. "Our honorable human visitor, I salute," she says loudly as she closes the gap. "Say, Captain, appreciate the fine arts, do you? To a cultural soiree, I'm headed, but busy with work, dearest Papa is, so vacant, his reserved seat is. Accompany me, would you?"`

Reviewer 3: Ah, i thought it was a megaphone or something. Sorry, I didn't read this one right.

> 			`Shortly after entering the spaceport, you observe an Arach in what you guess is a formal "dress," a delicate silk covering most of her body with each leg covered by its own embroidered skirt. She exclaims something loudly into a personal communication device before putting it away with a shove. Like everyone else who has momentarily been witness to this, you go to look away, but the Arach has cast their gaze around, seen you, and immediately begun to approach. "Our honorable human visitor, I salute," she says loudly as she closes the gap. "Say, Captain, appreciate the fine arts, do you? To a cultural soiree, I'm headed, but busy with work, dearest Papa is, so vacant, his reserved seat is. Accompany me, would you?"`

Reviewer 4: ```suggestion
			`Shortly after entering the spaceport, you observe an Arach in what you guess is a formal "dress," a delicate silk wrap covering most of her body with each leg covered by its own embroidered skirt. She exclaims something loudly into a personal communication device before putting it away with a shove. Like everyone else who has momentarily been witness to this, you go to look away, but the Arach has cast their gaze around, seen you, and immediately begun to approach. "Our honorable human visitor, I salute," she says loudly as she closes the gap. "Say, Captain, appreciate the fine arts, do you? To a cultural soiree, I'm headed, but busy with work, dearest Papa is, so vacant, his reserved seat is. Accompany me, would you?"`
```

> 			`Shortly after entering the spaceport, you observe an Arach in what you guess is a formal "dress," a delicate silk wrap covering most of her body with each leg covered by its own embroidered skirt. She exclaims something loudly into a personal communication device before putting it away with a shove. Like everyone else who has momentarily been witness to this, you go to look away, but the Arach has cast their gaze around, seen you, and immediately begun to approach. "Our honorable human visitor, I salute," she says loudly as she closes the gap. "Say, Captain, appreciate the fine arts, do you? To a cultural soiree, I'm headed, but busy with work, dearest Papa is, so vacant, his reserved seat is. Accompany me, would you?"`

Reviewer 5: "soirée" is normally spelled with a "é", which wouldn't be displayed in game. But I can't come up with a good alternative.

			choice
				`	"Is that some sort of event?"`
				`	"I'm busy right now, so I'll pass."`
					decline
			`	"No simple gathering, but a collection of plays by esteemed professionals, completed by a dedicated orchestra, the soiree is," she explains. "A celebration of Arachi folklore, it is, which in our finest outfits, us local patrons attend. Streamed live for others, the show also is, but compare to the live performance, it cannot."`
			choice
				`	"Alright, I'm interested. Where is this 'soiree' taking place?"`
				`	"I'm not a fan of plays, sorry."`
					decline

> 			`	You follow her around the central portion of the city to a building, which from the outside looks like some form of auditorium; there, some Arachi in uniform are seeing to the entries of the "patrons" of the event. "A human? Quite an exotic guest you've brought, lady Matlub," one of them says to your guide as she shows him a touchpad displaying two tickets. He verifies them and allows you two to enter. The inside reminds you of a human opera house, with a large stage in front of the dozens of rows of seats, and six levels of balconies on both sides. Matlub leads you to two excellent seats in one of the first few rows, and she helps you "adjust" the chair so that you're comfortably seated. Just before the stage is an area a few steps below the level of the seats, with a few Arachi musicians fine-tuning a variety of instruments. Up on the ceiling, and on all of the balconies, you see small cameras pointed at the stage to broadcast the event for those who didn't have the chance to attend it live.`

Reviewer 4: ```suggestion
			`	You follow her around the central portion of the city to a building which from the outside looks like some form of auditorium; there, some Arachi in uniform are seeing to the entries of the "patrons" of the event. "A human? Quite an exotic guest you've brought, lady Matlub," one of them says to your guide as she shows him a touchpad displaying two tickets. He verifies them and allows the two of you to enter. The inside reminds you of a human opera house, with a large stage in front of dozens of rows of seats, and six levels of balconies on both sides. Matlub leads you to two excellent seats in one of the first few rows, and she helps you "adjust" the chair so that you're more comfortably seated. Just before the stage is an area a few steps below the level of the seats, with a few Arachi musicians fine-tuning a variety of instruments. Up on the ceiling, and on all of the balconies, you see small cameras pointed at the stage to broadcast the event for those who didn't have the chance to attend it live.`
```

			`	As the rows and balconies fill up with more guests, you notice a few Kimek and Saryds amidst the masses of Arachi, most of them accompanying an Arach patron; a few are seemingly patrons themselves, coming in on their own. A great many of them seem to take notice of you as you look around, but your patron studiously ignores the attention. Once all seats are filled, the doors are closed, and an Arach comes to the stage. "Welcome you, esteemed guests, to this celebration of saga and song, our theater does," he says. "Make you wait any longer, I shall not, so for our first play, a timeless classic dear to all, I present: Moonstruck."`

> 			`	He quickly leaves the stage as the curtains open to reveal a rocky wall with two tunnels leading to the sides. The orchestra begins to play a slow, soft theme on accordion-like instruments, and a young Arach comes into the scene from the tunnel on the left. He hangs onto the rocky wall, a small lantern hanging from his abdomen and a large sack strapped to his back. He picks rocks from the corners of the wall, throwing most to the ground but placing a few inside the sack he carries. Once the sack is full, he enters the tunnel on the right, and the rocky wall is rotated to show the other side; there, the young miner is shown under the night sky smelting the ore he collected. As he looks to the top of the stage with his back facing the audience, the accordions grow quieter and a slow harp starts to play. A full moon begins to come into view at the top of the background sky.`

Reviewer 3: ```suggestion
			`	He quickly leaves the stage as the curtains open to reveal a rocky wall with two tunnels leading to the sides. The orchestra begins to play a slow, soft theme on accordion-like instruments, and a young Arach comes into the scene from the tunnel on the left. He hangs onto the rocky wall, a small lantern hanging from his abdomen and a large sack strapped to his back. The Arach then picks rocks from the corners of the wall, throwing most to the ground but placing a few inside the sack he carries. Once the sack is full, he enters the tunnel on the right, and the rocky wall is rotated to show the other side; there, the young miner is shown under the night sky smelting the ore he collected. As he looks to the top of the stage with his back facing the audience, the accordions grow quieter and a slow harp starts to play. A full moon begins to come into view at the top of the background sky.`
```

> 			`	He quickly leaves the stage as the curtains open to reveal a rocky wall with two tunnels leading to the sides. The orchestra begins to play a slow, soft theme on accordion-like instruments, and a young Arach comes into the scene from the tunnel on the left. He hangs onto the rocky wall, a small lantern hanging from his abdomen and a large sack strapped to his back. The Arach then picks rocks from the corners of the wall, throwing most to the ground but placing a few inside the sack he carries. Once the sack is full, he enters the tunnel on the right, and the rocky wall is rotated to show the other side; there, the young miner is shown under the night sky smelting the ore he collected. As he looks to the top of the stage with his back facing the audience, the accordions grow quieter and a slow harp starts to play. A full moon begins to come into view at the top of the background sky.`

Reviewer 4: ```suggestion
			`	He quickly leaves the stage as the curtains open to reveal a rocky wall with two tunnels leading to the sides. The orchestra begins to play a slow, soft theme on accordion-like instruments, and a young Arach comes into the scene from the tunnel on the left. He hangs onto the rocky wall, a small lantern hanging from his abdomen and a large sack strapped to his back. The Arach then picks rocks from the corners of the wall, throwing most to the ground but placing a few inside the sack he carries. Once the sack is full, he enters the tunnel on the right, and the rocky wall is rotated to show the other side; there, the young miner is shown under the night sky smelting the ore he collected. As he looks to the top of the stage with his back facing the audience, the accordions grow quieter, a slow harp starts to play, and a full moon comes into view at the top of the background sky.`
```

			`	The scene continues as a couple comes to the set, walking down the street. As they're about to lean in for what looks like a kiss, the Arach leaps from the building, landing on the bachelor with a thud. He steals the kiss, as the tubas and trumpets blast as loudly as they can. After a while, the authorities show up, only for the Arach to make a run for it down the street. The orchestra plays a tune that is as uncoordinated as his movements, and the curtains close. Not many in the audience manifest themselves, most sighing or getting ready to leave, save for the group that was laughing earlier who are now very vocal in announcing their appreciation for the play. "Know of subtetly, house Chamgar does not," Matlub says. "A shame it is, that tainted the soiree with this rivalry, they have."`
			`	The doors open once again as new patrons cycle in to watch the third play.`
			choice
				`	(Stay for the next play.)`
				`	(Leave.)`
					goto leaveearly
			`	Everyone takes their seats, and the Arach announcer heads to the stage again. "A cathartic tragedy, our third play of this evening is. For our next performance: 'Til Death."`

> 			`	The stage lights go dark as the curtains open to a single spotlight. On the right side of the stage it illuminates an old Arach getting dressed inside a small shack. He coughs, and a something akin to an organ begins to play a melancholic melody. Once he leaves, the shack is left in the dark as the spotlight follows him to the middle of the stage, where he makes his way carefully down an empty street. He is allowed to pass beyond the spotlight which goes dark as the perspective shifts and a second spotlight renews the focus. The left side of the stage is finally lit up, where the Arach starts tending to a garden by the entrance of a manor. The organ stops its weeping every once in a while, right when the Arach is coughing, only to resume once he's done. Leaving the manor, a richly dressed Arach points to one of the flowers in the garden, showing it to be shriveled and dead. He plucks it and throws it to the ground before leaving. The gardener picks up the flower, sticks it inside his clothes, and continues his work.`

Reviewer 3: ```suggestion
			`	The stage lights go dark as the curtains open to a single spotlight. On the right side of the stage it illuminates an old Arach getting dressed inside a small shack. He coughs, and a something akin to an organ begins to play a melancholic melody. Once he leaves, the shack is left in the dark as the spotlight follows him to the middle of the stage, where he makes his way carefully down an empty street. He is allowed to pass beyond the spotlight, which goes dark as the perspective shifts and a second spotlight renews the focus. The left side of the stage is finally lit up, where the Arach starts tending to a garden by the entrance of a manor. The organ stops its weeping every once in a while, right when the Arach is coughing, only to resume once he's done. Leaving the manor, a richly dressed Arach points to one of the flowers in the garden, showing it to be shriveled and dead. He plucks it and throws it to the ground before leaving. The gardener picks up the flower, sticks it inside his clothes, and continues his work.`
```

> 			`	The stage lights go dark as the curtains open to a single spotlight. On the right side of the stage it illuminates an old Arach getting dressed inside a small shack. He coughs, and a something akin to an organ begins to play a melancholic melody. Once he leaves, the shack is left in the dark as the spotlight follows him to the middle of the stage, where he makes his way carefully down an empty street. He is allowed to pass beyond the spotlight, which goes dark as the perspective shifts and a second spotlight renews the focus. The left side of the stage is finally lit up, where the Arach starts tending to a garden by the entrance of a manor. The organ stops its weeping every once in a while, right when the Arach is coughing, only to resume once he's done. Leaving the manor, a richly dressed Arach points to one of the flowers in the garden, showing it to be shriveled and dead. He plucks it and throws it to the ground before leaving. The gardener picks up the flower, sticks it inside his clothes, and continues his work.`

Reviewer 4: ```suggestion
			`	The stage lights go dark as the curtains open to a single spotlight. It illuminates an old Arach getting dressed inside a small shack on the right side of the stage. He coughs, and something akin to an organ begins to play a melancholic melody. Once he leaves, the shack is left in the dark as the spotlight follows him to the middle of the stage, where he makes his way carefully down an empty street. He is allowed to pass beyond the spotlight, which goes dark as the perspective shifts and a second spotlight renews the focus. The left side of the stage is finally lit up, where the Arach starts tending to a garden by the entrance of a manor. The organ stops its weeping every once in a while, right when the Arach is coughing, only to resume once he's done. Leaving the manor, a richly dressed Arach points to one of the flowers in the garden, showing it to be shriveled and dead. He plucks it and throws it to the ground before leaving. The gardener picks up the flower, sticks it inside his clothes, and continues his work.`
```

> 			`	Once his work is done, he heads back to his shack, the spotlights following him as he moves around the stage. Once inside, he looks inside a small box, taking a simple vase and placing the flower inside it. The stage goes dark as he goes to sleep and the piano stops, with only the occasional cough echoing throughout the theater. When the light comes to the shack again, the Arach slowly gets up and leaves for the gardening work again, pausing for a moment on the street part of the stage during a heavy coughing fit. Once he's done with the work and begins going back home, he starts coughing again, not letting up for more than a second at a time. During this the spotlights transition to the next section of the stage, and the piano plays a brief note. Once inside his shack, he collapses, knocking down the flower vase in the process. The piano plays a few slow notes, and the last spotlight goes out during the final note.`

Reviewer 3: Trying to make some sentences less unwieldy, and reduce word repetition.
```suggestion
			`	Once his work is done, he heads back to his shack, the spotlights following him as he moves around the stage. Once inside, he looks inside a small box, taking a simple vase and placing the flower inside it. The stage goes dark as he goes to sleep and the piano stops, with only the occasional cough echoing throughout the theater. When the light comes to the shack again, the Arach slowly gets up and again leaves for his job, pausing for a moment on the street part of the stage during a heavy coughing fit. Once he's done with the work and begins going back home, he starts coughing again, not letting up for more than a second at a time. During this the spotlights transition to the next section of the stage, and the piano plays a brief note. Once inside his shack, he collapses, knocking down the flower vase in the process. The piano plays a few slow chords, and the last spotlight goes out during the final note.`
```

> 			`	Once his work is done, he heads back to his shack, the spotlights following him as he moves around the stage. Once inside, he looks inside a small box, taking a simple vase and placing the flower inside it. The stage goes dark as he goes to sleep and the piano stops, with only the occasional coughing fit echoing throughout the theater. When the light comes to the shack again, the Arach slowly gets up and leaves for the gardening work again, pausing for a moment on the street part of the stage during a heavy coughing fit. Once he's done with the work and begins going back home, he starts coughing again, not letting up for more than a second at a time. During this the spotlights transition to the next section of the stage, and the piano plays a brief note. Once inside his shack, he collapses, knocking down the flower vase in the process. The piano plays a few slow notes, and the last spotlight goes out during the final note.`

Reviewer 3: ```suggestion
			`	Once his work is done, he heads back to his shack, the spotlights following him as he moves around the stage. Once inside, he looks inside a small box, taking a simple vase and placing the flower inside it. The stage goes dark as he goes to sleep and the piano stops, with only the occasional cough echoing throughout the theater. When the light comes to the shack again, the Arach slowly gets up and leaves for the gardening work again, pausing for a moment on the street part of the stage during a heavy coughing fit. Once he's done with the work and begins going back home, he starts coughing again, not letting up for more than a second at a time. During this the spotlights transition to the next section of the stage, and the piano plays a brief note. Once inside his shack, he collapses, knocking down the flower vase in the process. The piano plays a few slow notes, and the last spotlight goes out during the final note.`
```

			`	You hear the curtains closing, and the lights of the theater turn back on again as the audience is taken by a round of applause. Once they've settled, the announcer goes up to the stage again. "And with that, concluded, our soiree is. Thank you all for gracing us with your presence, I, in name of the theater, do." He lowers his head in a bow and leaves the stage again as nearly all of the patrons in the audience get up and head for the exit, this time with nobody else coming in to take their seats. At the same time, the orchestra starts packing up their instruments. There are, however, a few who remain in their seats, including Matlub.`
			choice
				`	(Stay.)`
					goto end
				`	(Leave.)`
			label leaveearly
			`	You thank Matlub for inviting you here, and while she doesn't seem to pay you much mind, you can feel the gazes of other Arachi patrons on you, whispering among themselves as you leave. A few minutes after leaving, you're back at your ship.`
				decline
			label end

> 			`	The theater workers leave one after the other, leaving you and the other guests with an empty stage. After a while, the lights are turned off. A couple minutes later, however, you hear the curtains open, as some violins start to play in unison. Their music doesn't come from where the orchestra was seated, but as if they were all over the theater, behind you and even up on the balconies. A strong light comes from the stage, most of it blocked by a tall, dark figure. However, it isn't an actor, instead seemingly made out of wooden parts kept up by thin wires you can barely spot when looking at the top of the stage. The violins' song is disrupted by dull, successive drum hits, at which point the figure moves, with its "arms" opening to the sides; it becomes even taller as it stands up. It is a humanoid creature, and it begins to slowly move around the stage, as if dancing in a pendulum. Cellos join in with the violin screams, and are followed by a choir, though you cannot understand what they said. You do recognize, however, that it isn't being sung in any language of the Coalition.`

Reviewer 4: ```suggestion
			`	The theater workers leave one after the other, leaving you and the other guests with an empty stage. After a while, the lights are turned off. A couple minutes later, however, you hear the curtains open, as some violins start to play in unison. Their music doesn't come from where the orchestra was seated, but as if they were all over the theater, behind you and even up on the balconies. A strong light comes from the stage, most of it blocked by a tall, dark figure. It isn't an actor, however, instead seeming to be made out of wooden parts kept up by thin wires you can barely spot when looking at the top of the stage. The violins' song is disrupted by dull, successive drum hits, at which point the figure moves, with its "arms" opening to the sides; it becomes even taller as it stands up. It is a humanoid creature, and it begins to slowly move around the stage, as if dancing in a pendulum. Cellos join in with the violin screams, and are followed by a choir, though you cannot understand their words. You do recognize, however, that it isn't being sung in any language of the Coalition.`
```

> 			`	The creature appears to get closer to the audience, growing in size, and as the choir and instruments become more intense, it swings one of its arms upward and removes the wires. Despite this, it stays upright and continues to move. It brings both arms close and strikes its own head to form two small holes from which the background light shines through. Then, it falls to its knees, with one arm reaching for the audience, as the light behind it turns off and the choir and music subside.`

Reviewer 4: ```suggestion
			`	The creature appears to get closer to the audience, growing in size, and as the choir and instruments become more intense, it swings one of its arms upward and removes the wires. Despite this, it stays upright and continues to move. It brings both arms close and strikes its own head to form two small holes which the background light shines through. Then, it falls to its knees, with one arm reaching for the audience, as the light behind it turns off and the choir and music subside.`
```

			choice
				`	"Alright, I'm interested. Where is this 'soiree' taking place?"`
				`	"I'm not a fan of plays, sorry."`
					decline
			`	You follow her around the central portion of the city to a building which from the outside looks like some form of auditorium; there, some Arachi in uniform are seeing to the entries of the "patrons" of the event. "A human? Quite an exotic guest you've brought, lady Matlub," one of them says to your guide as she shows him a touchpad displaying two tickets. He verifies them and allows the two of you to enter. The inside reminds you of a human opera house, with a large stage in front of dozens of rows of seats, and six levels of balconies on both sides. Matlub leads you to two excellent seats in one of the first few rows, and she helps you "adjust" the chair so that you're more comfortably seated. Just before the stage is an area a few steps below the level of the seats, with a few Arach musicians fine-tuning a variety of instruments. Up on the ceiling, and on all of the balconies, you see small cameras pointed at the stage to broadcast the event for those who didn't have the chance to attend it live.`

> 			`	As the rows and balconies fill up with more guests, you notice a few Kimek and Saryds amidst the masses of Arachi, most of them accompanying an Arach patron; a few are seemingly patrons themselves, coming in on their own. A great many of them seem to take notice of you as you look around, but your patron studiously ignores the attention. Once all seats are filled, the doors are closed, and an Arach comes to the stage. "Welcome you, esteemed guests, to this celebration of saga and song, our theater does," he says. "Make you wait any longer, I shall not, so for our first play, a timeless classic dear to all, I present: Moonstruck."`

Author: Something like the latest commit?

> 			`	As the rows and balconies fill up with more guests, you notice a few Kimek and Saryds amidst the masses of Arachi, most of them accompanying an Arach patron; a few are seemingly patrons themselves, coming in on their own. A great many of them seem to take notice of you as you look around, but your patron studiously ignores the attention. Once all seats are filled, the doors are closed, and an Arach comes to the stage. "Welcome you, esteemed guests, to this celebration of saga and song, our theater does," he says. "Make you wait any longer, I shall not, so for our first play, a timeless classic dear to all, I present: Moonstruck."`

Reviewer 6: Perfect.
